cmake/MacroConnext.cmake

```diff
@@ -12,17 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-##############################################################################
```

Comment:
Remove this file altogether.

Reply:
It builds and tests flawlessly: file removed.